src/App.scss

```diff
@@ -0,0 +1,134 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 40vmin;
+  pointer-events: none;
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+
+.tag__save__header {
+  position: fixed;
+  top:0;
+  background-color: white;
+}
+
+.container {
+  height: 100vh;
+  width: 100vw;
+  text-align: center;
+}
+
+.error{
+  color: red;
+  font-size: 4vw;;
+}
+
+.success {
+  color: green;
+  font-size: 2vw;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.select__index{
+  height: 18vw;
+  width: 24vw;
+}
+
+.image__index{
+  height: 40vw;
+  width: 24vw;
+  overflow: auto;
+  white-space: nowrap;
+}
+
+.menu__section {
+  width: 25vw;
+  display: inline-block;
+}
+
+.selected__image{
```

Comment:
if we organized components into their own directories, it can have a local stylesheet. this will be more and more powerful and useful as we add more components and styles, as at this rate this main `App.css` file will rapidly grow and can become difficult to maintain. consider a directory that's generally like:

```
├── ImageViewDisplay
     ├── index.js
     ├── styles.css
     └──  tests/
            └──  index.test.js
```

then in the `index.js` file, we can add a local import to its own stylesheet:

```
import './styles.css;
```

that will contain css for only that component. nowadays, css-in-in-js solutions are popular but generally the pattern of "initially keep things as local as possible, then abstract to the next level if used in multiple places" applies anywhere.

Reply:
ohhhh that makes a lot of sense. I was having issues was the organization already and found myself using the same class selector multiple times in the stylesheets.